modules/images-triggering-updates-imagestream-changes-kubernetes-cli.adoc

```diff
@@ -4,11 +4,9 @@
 
 
 [id="images-triggering-updates-imagestream-changes-kubernetes-cli_{context}"]
-= Triggering Kubernetes resources manually
+= Setting the Image Trigger on Kubernetes resources
 
-You can manually trigger Kubernetes resources by entering the `oc set triggers` command.
-
-When adding an image trigger to deployments, you can use the `oc set triggers` command. For example, the example command in this procedure adds an image change trigger to the deployment named `example` so that when the `example:latest` image stream tag is updated, the `web` container inside the deployment updates with the new image value.
+When adding an image trigger to deployments, you can use the `oc set triggers` command. For example, the sample command in this procedure adds an image change trigger to the deployment named `example` so that when the `example:latest` image stream tag is updated, the `web` container inside the deployment updates with the new image value.  This command will set the correct `image.openshift.io/triggers` annotation on the deployment resource.
```

Comment:
Double space before "This command"

s/This command will set.../This command sets...